### DISCUSSION TITLE
Cannot edit predictions (double-click not working)

### DISCUSSION THREAD

**Discussion by lisadiez** [talmolab/sleap#1063] [Help!]
Hello,

We're trying to correct predictions but cannot turn predicted instances into editable instances. Double-clicking does not seem to be working.

Could someone please help with this?

Here is a video of what happens:

https://user-images.githubusercontent.com/116014547/204333686-763a3c12-cd62-4316-b92e-d3f460af0370.mov

And this is the command line output:

<img width="840" alt="commandline_output" src="https://user-images.githubusercontent.com/116014547/204333886-5bac3c61-5ba1-401a-8f80-9a0466592195.png">

Thank you in advance!
Lisa

**Comment by roomrys** [comment]
Hi Lisa,

It seems you have run into one of our past bugs #713 which allowed users to load multiple skeletons into the project through the Load Skeleton button. Currently SLEAP can only handle a single skeleton in use, but you had two skeletons in use. Luckily, there were just 3 labeled frames that used the second skeleton - I removed these labeled frames from the project, namely:
```
In [38]: lfs_wrong_skeleton
Out[38]:
[LabeledFrame(video=MediaVideo('2020_09_07_06_45_47_719700_compressed_cropped.mp4'), frame_idx=5505, instances=1),
 LabeledFrame(video=MediaVideo('2020_09_05_19_32_11_797812_compressed_cropped.mp4'), frame_idx=3540, instances=2),
 LabeledFrame(video=MediaVideo('2020_09_05_19_32_11_797812_compressed_cropped.mp4'), frame_idx=3540, instances=2)]
```
and this has resolved the issue.

Did you use the Load Skeleton button or merge the project at some point? Also which SLEAP version are you running?

Thanks,
Liezl

**Reply by lisadiez** [reply]
Thank you so much Liezl! 
We didn't use the Load Skeleton button, but we did merge the predictions to the project (we run training and predictions on the command line because we're using the bottom up with ID model). 
I'm pretty sure we're running v1.1.5
